### PR TITLE
Fix GH url for sample data download

### PR DESCRIPTION
**Pull request by Cadair**
This is an alternative to #2197

### CODE REVIEW

File: sunpy/data/_sample.py [hunk 1]
```diff
@@ -49,7 +49,8 @@
     "EIT_195_IMAGE": "eit_l1_20110607_203753.fits",
     "RHESSI_IMAGE": "hsi_image_20110607_063300.fits",
     "CALLISTO_SPECTRUM": "BIR_20110607_062400_10.fit",
-    "RHESSI_EVENT_LIST": "hsi_calib_ev_20020220_1106_20020220_1106_25_40.fits",
+    # Not in the sample-data repo
+    # "RHESSI_EVENT_LIST": "hsi_calib_ev_20020220_1106_20020220_1106_25_40.fits",
```
Comment (Cadair) [inline sunpy/data/_sample.py]: @ehsteve should this be here?! what's going on?

**Comment by nabobalis** [comment]
```/home/travis/build/sunpy/sunpy/examples/timeseries_example.py failed to execute correctly
AttributeError: module 'sunpy.data.sample' has no attribute 'RHESSI_TIMESERIES'
```

Change in examples needed with new test data?

**Comment by Cadair** [comment]
Omg this fiasco is about to make me turn on the ci must pass thing on the whole repo.

File: .travis.yml [hunk 2]
```diff
@@ -64,7 +68,7 @@ matrix:
          - python: 3.6
            env: SETUP_CMD='build_docs -w'
                 EVENT_TYPE='push pull_request cron'
-                SPHINX_VERSION='>1.6'
+                SPHINX_VERSION='<1.6'
```
Comment (nabobalis) [inline .travis.yml]: Booooooooooooooooooooooooooooooooooooooooooo

**Pull request merged**